source/bson-binary-vector/bson-binary-vector.md

```diff
@@ -0,0 +1,103 @@
+# BSON Binary Subtype 9 - Vector
+
+- Status: Pending
+- Minimum Server Version: N/A
+
+______________________________________________________________________
+
+## Abstract
+
+This document describes the addition of a new subtype to the Binary BSON type. This subtype is used for efficient
```

Comment:
"new" is always relative, I imagine this document should be fairly static in time, i.e. still be relevant as-is in 5 years if we don't find a need to make any changes to it?

Reply:
That's a great call. I'll update thinking not of the first users, but of those in 5 years.

Reply:
Updated